src/components/UI/Button/Button.test.js

```diff
@@ -16,12 +16,16 @@ describe("Button component", () => {
     expect(button.toJSON()).toMatchSnapshot();
   });
 
-  it("shows rounded button", () => {
-    const button = renderer.create(
-      <Button className="rounded">Test Button</Button>
-    );
+  it("shows rounded button type", () => {
+    const button = renderer.create(<Button type="rounded">Test Button</Button>);
     const tree = button.toJSON();
-    expect(tree.props.className).toContain("rounded");
+    expect(tree.props.type).toContain("rounded");
+  });
+
+  it("shows orange button colour", () => {
+    const button = renderer.create(<Button color="orange">Test Button</Button>);
+    const tree = button.toJSON();
+    expect(tree.props.color).toContain("orange");
```

Comment:
this test does nothing =>
- we create a button with a prop color set to orange, and then we test that the prop color is set to orange
- we need to test the expected result => css color is set to orange for instance :)